OSDOCS-5178:Allow users to create more than one sc (#55822)
openshift/openshift-docs: modules/dynamic-provisioning-change-default-class.adoc

gp2 (default)        kubernetes.io/aws-ebs <1>
standard             kubernetes.io/aws-ebs
----
<1> `(default)` indicates the default storage class.

. Make the desired storage class the default by setting the `storageclass.kubernetes.io/is-default-class` annotation to `true`:
+
[source,terminal]
----
$ oc patch storageclass standard -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
----
+
[NOTE]
====

> It is acceptable to have multiple default storage classes for a short time. However, you should ensure that eventually only one default storage class exists.

Reviewer: I would highlight what happens in that case (the most recently created SC is used)

// add xref to multi/no default SC module
====

> . Change the value of the `storageclass.kubernetes.io/is-default-class` annotation to `false` for the previous default storage class:

Reviewer 2: ```suggestion
. Change the value of the `storageclass.kubernetes.io/is-default-class` annotation to `false` by running the following command:
```

gp3 (default)        kubernetes.io/aws-ebs <1>
standard             kubernetes.io/aws-ebs
----
<1> `(default)` indicates the default storage class.

. Make the desired storage class the default by setting the `storageclass.kubernetes.io/is-default-class` annotation to `true`:
+
[source,terminal]
----
$ oc patch storageclass standard -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "true"}}}'
----
+
[NOTE]
====

> It is acceptable to have multiple default storage classes for a short time. However, you should ensure that eventually only one default storage class exists.

Reviewer: I would highlight what happens in that case (the most recently created SC is used)

Reviewer 3: Yeah, according to latest sc time stamp.

Author: @lpettyjo remember to add xref to multi/no sc default sc module in note.

Author: Done.

// add xref to multi/no default SC module
====

. Change the value of the `storageclass.kubernetes.io/is-default-class` annotation to `false` for the previous default storage class:
+
[source,terminal]
----

> $ oc patch storageclass gp2 -p '{"metadata": {"annotations": {"storageclass.kubernetes.io/is-default-class": "false"}}}'

Author: Good catch. Thanks!
